PostgresExtended/bin/extensions/pgosm/install.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+apt-get update
```

Comment:
Make sense to move `update` and `clean`  commands up to PostgresExtended/bin/install.sh

Reply:
ack